app/views/books/index.html.erb

```diff
@@ -0,0 +1,6 @@
+<h1>All Books</h1>
+<h3><a href="/books/new">New Book</a></h3>
+
+<% @books.each do |book| %>
+<h3><a href="/books/<%= book.id %>"><%= book.title %></a></h3>
+<% end %>
```

Comment:
Great catch, I really like that this links to the book show page. It makes more sense.